### PR TITLE
Feature/responsive sidebar

### PR DESCRIPTION
**Pull request by joecohens**
This is working on the main menu.

Need to make the same for the sub-menu and implement animate.css

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Conflicts already?

**Comment by joecohens** [comment]
Yea sorry I'm not in town since yesterday I just push it has a reference. I'll try to finish it tonight :)

**Comment by jbrooksuk** [comment]
:+1:

**Comment by joecohens** [comment]
I've rebased and compiled with --production. Responsive it's working on main menu but still needs some work on the settings sub menu but this ir ready to merge. @jbrooksuk Let me know what you think!

File: bower.json [hunk 1]
```diff
@@ -1,16 +1,18 @@
+
```
Comment (GrahamCampbell) [inline bower.json]: extra line needs removing

**Comment by GrahamCampbell** [comment]
Also, `--production` shouldn't be required anymore. @jbrooksuk enabled it by default.

File: bower.json (hunk 1 above)
Comment (joecohens) [inline bower.json]: Thanks! Missed that on the rebase.

**Comment by joecohens** [comment]
By default won't let us debug with out source maps on dev.. But I guess it's ok for now.

**Comment by jbrooksuk** [comment]
@joecohens if you want to debug, then run `gulp --production=false` I think that'll work.

**Comment by joecohens** [comment]
@jbrooksuk Thanks!

**Comment by jbrooksuk** [comment]
The semi-responsive looks great!

**Comment by jbrooksuk** [comment]
Although there could be less margin below the user profile.

**Comment by joecohens** [comment]
Thanks! I think so too, also it would be cool to add some kind of animation.

**Comment by jbrooksuk** [comment]
:+1: for the animation, but not too obtrusive.

Also, what about mobile? The nav disappears.

**Comment by joecohens** [comment]
No just like size of the icons it would look fancy haha. And you're right I have to add a button to show on mobile.

**Comment by jbrooksuk** [comment]
I love the mini-sidebar :)

**Comment by joecohens** [comment]
:)

**Pull request merged**